autoload/clang_format.vim

```diff
@@ -109,7 +109,11 @@ endfunction
 
 function! clang_format#is_invalid()
     if !exists('s:command_available')
-        if ! executable(g:clang_format#command)
+	    " if running on windows and the path has spaces it needs to be quoted,
+		" but executable() get confused by the quotes, so we remove them
+	    if has("win32") && !executable(split(g:clang_format#command, '"')[0])
+		    return 1
+        elseif !has("win32")  && !executable(g:clang_format#command)
```

Comment:
- Conditions of `if` are complicated.  I think it's good to separate them to multiple `if` statements
- Indentation looks collapsed

```vim
if has('win32') || has('win64')
    if !executable(split(...))
        return 1
    endif
elseif !executable(g:clang_format#command)
    return 1
endif
```